src/main/groovy/org/arquillian/spacelift/gradle/SpaceliftExtension.groovy

```diff
@@ -109,30 +115,38 @@ class SpaceliftExtension {
         return "SpaceliftExtension" + (project.buildFile ? "(${project.buildFile.canonicalPath})" : "")
     }
 
-
     /**
      * If property was not found, try to check content of container for resolution
      * @param name Name of the property, can be any object defined in DSL
      * @return
      */
-     def propertyMissing(String name) {
-
+    def propertyMissing(String name) {
         // try all containers to find resolution in particular order
         //order here defines order of reference in case reference to the same object is found, laters are ignored
         def object, objectType
-        for(def container : ([installations, tests, tools, profiles])) {
+
+        def containers = [configuration, installations, tests, tools, profiles]
+        if (project.hasProperty("selectedProfile")) {
```

Comment:
Can you please add this marker there? https://github.com/arquillian/arquillian-spacelift-gradle-plugin/issues/50